### DISCUSSION TITLE
openai_api_demo/api_server.py中的/v1/chat/completions接口

### DISCUSSION THREAD

**Discussion by pengroc** [zai-org/ChatGLM3#1179] [Q&A] [answered]
接口能否支持history历史会话的传递？现在调用接口是一问一答形式，能像basic_demo/web_demo_gradio.py记录历史对话吗?

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
openai不都是这样传入历史的，message加入历史就行了，怎么工程设计需要自己做，这里只模仿了接口
